src/com/alevelhw/hw7/action/Work.java

```diff
@@ -0,0 +1,52 @@
+package com.alevelhw.hw7.action;
+
+import com.alevelhw.hw7.model.Animal;
+
+public class Work {
+    private int workCount;
+
+    public void doAction(Animal animal) {
+        final int allowedWorkingAge = 5;
+        final int dropHunger = 4;
+        final int dropHappy = 2;
+        final int minBoundAddMoney = 5;
+        final int maxBoundAddMoney = 20;
+        final int quantityWorkForUpAge = 6;
+        final int upAge = 1;
```

Comment:
It is better to make the fields of the class, since each time the method is called, we create the values anew